src/modules/cmdpal/Microsoft.CmdPal.UI.ViewModels/Commands/MainListPage.cs

```diff
@@ -149,13 +150,13 @@ private bool ActuallyLoading()
     // _always_ show up first.
     private int ScoreTopLevelItem(string query, IListItem topLevelOrAppItem)
     {
-        if (string.IsNullOrWhiteSpace(query))
+        if (string.IsNullOrEmpty(query))
         {
             return 1;
         }
 
         var title = topLevelOrAppItem.Title;
-        if (string.IsNullOrEmpty(title))
```

Comment:
... if there was a non-whitespace filter, we'd then remove items with an empty title here